src/app/workspace/[workspaceId]/meeting-notes/page.tsx

```
@@ -0,0 +1,661 @@
+														<CheckSquare className="w-3.5 h-3.5 text-emerald-500" />{" "}
+														Action Items
+													</h4>
+													<span className="text-[10px] font-bold bg-emerald-100 text-emerald-700 px-2 py-0.5 rounded-full">
+														{genItems.length} tasks
+													</span>
+												</div>
+												<div className="space-y-2">
+													{genItems.map((task: any, i: number) => (
+														<div
+															className="bg-white dark:bg-zinc-900 border border-gray-100 dark:border-zinc-800 p-4 rounded-xl hover:border-emerald-200 dark:hover:border-emerald-800 transition-colors group"
+															key={i}
```

Biome (2.4.15)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)